src/v/cluster/scheduling/leader_balancer_greedy.h

```diff
@@ -59,6 +59,10 @@ class greedy_balanced_shards final : private leader_balancer_strategy {
           });
     }
 
+    double max_error() const final {
```

Comment:
Is this unused?

Reply:
Yep, was going to use it for something. However, it didn't work out. Will remove.